compiler/test/src/org/jetbrains/dukat/compiler/tests/OutputTests.kt

```diff
@@ -64,8 +64,8 @@ abstract class OutputTests {
         val translated = output(tsPath, getTranslator())
 
         assertEquals(
-                translated,
-                File(ktPath).readText().trimEnd()
+                File(ktPath).readText().trimEnd(),
+                translated
```

Comment:
http://junit.sourceforge.net/junit3.8.1/javadoc/junit/framework/Assert.html#assertEquals(java.lang.Object,%20java.lang.Object)

expected should go first, otherwise test results would be confusing

Reply:
It's a matter of taste  - in my opinion it's more convenient to see actual on the left and expected on the right. It's debatable we can discuss this later - however in the framework of this particular PR it's just not relevant, so in this PR let's leave it as it was.